sys/compat/freebsd32/syscalls.master

```diff
@@ -491,60 +500,73 @@
 256	AUE_AIO_WRITE	STD	{ int freebsd32_aio_write( \
 				    struct aiocb32 *aiocbp); }
 257	AUE_LIO_LISTIO	STD	{ int freebsd32_lio_listio(int mode, \
-				    struct aiocb32 * const *acb_list, \
+				    uint32_t *acb_list, \
 				    int nent, struct sigevent32 *sig); }
-258	AUE_NULL	UNIMPL	nosys
-259	AUE_NULL	UNIMPL	nosys
-260	AUE_NULL	UNIMPL	nosys
-261	AUE_NULL	UNIMPL	nosys
-262	AUE_NULL	UNIMPL	nosys
-263	AUE_NULL	UNIMPL	nosys
-264	AUE_NULL	UNIMPL	nosys
-265	AUE_NULL	UNIMPL	nosys
-266	AUE_NULL	UNIMPL	nosys
-267	AUE_NULL	UNIMPL	nosys
-268	AUE_NULL	UNIMPL	nosys
-269	AUE_NULL	UNIMPL	nosys
-270	AUE_NULL	UNIMPL	nosys
-271	AUE_NULL	UNIMPL	nosys
-272	AUE_O_GETDENTS	COMPAT11 { int freebsd32_getdents(int fd, char *buf, \
+258	AUE_NULL	RESERVED
```

Comment:
We might collapse these to the range syntax for symmetry in syscalls.master pre-commit (noted that it is convenient for the review, indeed :-))